### DISCUSSION TITLE
getSessionDetails removed from v7 to v8?

### DISCUSSION THREAD

**Discussion by Auto81** [appium/java-client#1764] [Q&A]
In the process of upgrading from java-client v7.5.1 to v8.2.0

Previously was able to get the bundleId for the session on iOS using 

`((MobileDriver)driver).getSessionDetails().get("bundleId").toString()`

However after upgrading getSessionDetails() doesn't seem to be available. Have looked at the release notes and upgrade/migration docs but can't find any reference.

Have tried all the below options. None of them have getSessionDetails() available

```
      ((IOSDriver)driver).getSessionDetails();
      ((AppiumDriver)driver).getSessionDetails();
      ((WebDriver)driver).getSessionDetails();
      ((RemoteWebDriver)driver).getSessionDetails();
```

**Comment by mykola-mokhnach** [comment]
getSessionDetails has been deprecated as it is not a part of W3C spec. You could use https://github.com/appium/appium-xcuitest-driver#mobile-activeappinfo instead
